packages/nimble-components/src/patterns/anchor/tests/target-docs.mdx

```diff
@@ -0,0 +1,5 @@
+In general, when adding an anchor element to a web application, the anchor should not have a `target` configured on it to control how the link is opened.
```

Comment:
I think specifically we are concerned about a target configured for `_blank` with the intention of opening in a new window or tab.

The other target configurations are use-case specific and not really a big deal / worth calling out (i.e. `_top` / `_parent` are for iframe things and other names are for targeting specific named windows which is real uncommon nowadays).